contracts/ServiceNodeRewards.sol

```diff
@@ -755,11 +797,40 @@ contract ServiceNodeRewards is Initializable, Ownable2StepUpgradeable, PausableU
     /// the owner.
     /// @param newMax The new maximum non-signer threshold
     function setBLSNonSignerThresholdMax(uint256 newMax) public onlyOwner {
-        require(newMax > 0, "The new BLS non-signer threshold must be non-zero");
+        if (newMax <= 0)
+            revert PositiveNumberRequirement();
         blsNonSignerThresholdMax = newMax;
         emit BLSNonSignerThresholdMaxUpdated(newMax);
     }
 
+    /// @notice Max Claim amount to use in the Exponential Moving Average check
+    /// before allowing the user to claim. If the claimed amount over 24 hours
+    /// exceeds this then the claim will fail
+    /// @param newMax The new maximum non-signer threshold
+    function setClaimEMAThreshold(uint256 newMax) public onlyOwner {
+        if (newMax <= 0)
+            revert PositiveNumberRequirement();
+        claimEMAThreshold = newMax;
+        emit ClaimEMAThresholdUpdated(newMax);
+    }
+
+    /// @notice Updates the Exponential Moving Average (EMA) with a new value
+    /// @dev This function calculates a time-weighted EMA using the provided new value and current time
+    /// @param ema The TimeWeightedEMA struct to update
+    /// @param newValue The new value to incorporate into the EMA.
+    /// @param currentTime The current timestamp, usually `block.timestamp`
+    /// @custom:formula EMA = EMA_old + α * (newValue - EMA_old)
```

Comment:
This formulation of the mathematical equation makes it a bit more obvious how EMA works: basically it's a weighted sum of the previous value and the new value, with $\alpha$ being the weight for the new value:
```suggestion
    /// @custom:formula EMA = (1-α) * EMA_old + α * newValue
```